.github/workflows/gcc-compilation.yml

```diff
@@ -0,0 +1,64 @@
+name: GNU GCC intrinsic test cases compilation
+on:
+  schedule:
+  - cron: "0 18 * * 0-6"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Prerequisites
+        run:
+          sudo apt-get install autoconf automake autotools-dev curl python3 python3-pip
+          libmpc-dev libmpfr-dev libgmp-dev gawk build-essential bison flex texinfo gperf
+          libtool patchutils bc zlib1g-dev libexpat-dev ninja-build git cmake libglib2.0-dev
+          dejagnu
+      - name: Download GCC
+        run: |
+          cd ..
+          rm -rf riscv-gnu-toolchain
+          git clone --recursive https://github.com/riscv-collab/riscv-gnu-toolchain.git
+          cd ./riscv-gnu-toolchain/gcc
+          git remote update
+          git reset --hard origin/master
+      - name: Run auto generated test cases
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+          cd ../riscv-gnu-toolchain
+          ./configure --prefix="$PWD/install" \
+            --with-arch=rv64gcv \
+            --with-abi=lp64d \
+            --with-isa-spec=20191213 \
+            --with-sim=qemu
+          mkdir -p gcc/gcc/testsuite/gcc.target/riscv/rvv/gcc-auto-generated/gnu-api-tests
```

Comment:
Can we have this as a target under `Makefile` and call the target in the Github workflow? The main motivation for this is for users to also easily reproduce the CI commands with `make`.

Reply:
It takes about 3 hours to run gcc testuite (download and test). Is such a Makefile target acceptable?
@eopXD

Reply:
It takes 1 hour and 20 minutes without downloading the code
@eopXD

Reply:
Why not, since we will be calling them in the Github Action.

I am not indicating we add this to target of `all`, but to collect them as an action and allow the users to reproduce test verification locally.